WordPress/src/main/java/org/wordpress/android/datasets/AccountTable.java

```diff
@@ -0,0 +1,90 @@
+package org.wordpress.android.datasets;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import org.wordpress.android.WordPress;
+import org.wordpress.android.models.Account;
+import org.wordpress.android.util.SqlUtils;
+
+public class AccountTable {
+    // Warning: the "accounts" table in WordPressDB is actually where blogs are stored.
+    private static final String ACCOUNT_TABLE = "tbl_accounts";
+
+    private static SQLiteDatabase getReadableDb() {
+        return WordPress.wpDB.getDatabase();
+    }
+    private static SQLiteDatabase getWritableDb() {
+        return WordPress.wpDB.getDatabase();
+    }
+
+    public static void createTables(SQLiteDatabase db) {
+        db.execSQL("CREATE TABLE " + ACCOUNT_TABLE + " ("
+                + "	local_id                INTEGER PRIMARY KEY DEFAULT 0,"
+                + "	is_wordpress_com_user   INTEGER DEFAULT 0,"
+                + "	user_name	            TEXT,"
+                + "	user_id	                INTEGER DEFAULT 0,"
+                + "	display_name	        TEXT,"
+                + "	profile_url 	        TEXT,"
+                + " avatar_url   	        TEXT,"
+                + " primary_blog_id         INTEGER DEFAULT 0,"
+                + " site_count              TEXT,"
+                + " visible_site_count      TEXT,"
```

Comment:
It looks like `site_count` and `visible_site_count` are integers - why are they `TEXT` instead of `INTEGER DEFAULT 0`?